### DISCUSSION TITLE
Need to know about Change section is visibility condition for Contract notice

### DISCUSSION THREAD

**Discussion by ghost** [OP-TED/eForms-SDK#318] [Q&A] [answered]
Hi,

The query is related to **Change** section which is part of Contract notice ([subtype#16](https://github.com/OP-TED/eForms-SDK/blob/develop/notice-types/16.json)). 

Once the parent notice (e.g. Contract notice) is published, I want to correct the few details. I tried in preview environment but when I create the Change notice on top of it, I don't see the Change section in the list. See below fig.

<img width="1525" alt="Screenshot 2023-01-27 at 3 42 55 PM" src="https://user-images.githubusercontent.com/122265333/215061931-7ad0a118-211e-4fe1-868c-f381289a9373.png">

Query is - What are the conditions in which that change section will be visible on the UI?

Let me know if more information is required.

Thanks,
Chandresh Parmar

**Comment by YvesJo** [comment, marked as answer]
Hi Chandresh,
Make sure the notice to be updated is already in the "PUBLISHED" state. Also for this non technical question I suggest you to get in touch directly with the Helpdesk: ted@publications.europa.eu

**Reply by ghost** [reply]
Thanks. I created fresh notice with latest SDK version and Change section is appearing now. Looks like during version upgrade the Change section is disappearing.
